Makefile

```diff
@@ -217,7 +217,7 @@ check-fast: lint $(PYPY) $(PY36) $(TOX)
 check-rst: $(RSTLINT) $(FLAKE8)
 	$(RSTLINT) CONTRIBUTING.rst README.rst
 	$(RSTLINT) guides/*.rst
-	$(FLAKE8) --select=W191,W291,W292,W293,W391 *.rst hypothesis-python/docs/*.rst
+	$(FLAKE8) --select=W191,W291,W292,W293,W391 *.rst hypothesis-python/*.rst hypothesis-python/docs/*.rst
```

Comment:
Whoops. I think the new code would have handled this automatically (it finds all RST files and excludes) but still good to have this fixed now!